aztk/client/base/helpers/create_user_on_node.py

```diff
@@ -0,0 +1,42 @@
+from datetime import datetime, timedelta, timezone
+
+import azure.batch.models as batch_models
+import azure.batch.models.batch_error as batch_error
+
+from aztk import models
+from aztk.utils import get_ssh_key
+
+
+def __create_user(self, pool_id: str, node_id: str, username: str, password: str = None, ssh_key: str = None) -> str:
+    """
+        Create a pool user
+        :param pool: the pool to add the user to
```

Comment:
Wrong format of args

Reply:
these are the old deprecated methods, I'm going to leave the docstrings as is (since they will be removed soon).

All of the new user-facing functions have docstrings in the proper format.